addons/project/views/project_views.xml

```diff
@@ -938,6 +952,18 @@
             </field>
         </record>
 
+        <record id="view_project_config_kanban" model="ir.ui.view">
+            <field name="name">project.kanban.inherit.config.project</field>
+            <field name="model">project.project</field>
+            <field name="mode">primary</field>
+            <field name="inherit_id" ref="view_project_kanban"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[hasclass('o_project_kanban_boxes')]" position="attributes">
+                    <attribute name="class">o_project_kanban_config_boxes</attribute>
```

Comment:
Why do we need to add a class?

Reply:
@IT-Ideas Previously, the action was handled by javascript which was bound to
'o_projectkanban_boxes," which is responsible for opening project task
kanban view.
The action attribute is now available, so I adapted the code accordingly.